mesonbuild/envconfig.py

```diff
@@ -64,7 +64,9 @@
     'wasm32',
     'wasm64',
     'x86',
-    'x86_64'
+    'x86_64',
+    'ip30',
+    'ip35'
```

Comment:
I don't think this is right, I think these should be caught and converted to mips or mips64. These are for cpu families, not specific cpus. Think x86 vs i486, i586, i686.

Reply:
Unless I'm completely misremembering what was in the old SGI machines?

Reply:
No, this is a good call. I'll fix it.

Reply:
The `detect_cpu_family` function should be updated to handle these instead.